docs/Performance/promise.md

```diff
@@ -0,0 +1,195 @@
+    // success handler
+    console.log(result.data, result.response);
+  })
+  .catch(function (error) {
+    // error handler
+    console.error(error);
+  });
+```
+
+### Refactor the Promisified OData call
+
+Now, let's make the OData call more robust for everyday usage:
+
+* hand over the whole entity as object and construct the key from its data
+* provide an option to add additional data like filters to the call (mParameters as described in the [documentation](https://ui5.sap.com/#/api/sap.ui.model.odata.v2.ODataModel%23methods/read))
+* make sure its not possible to provide success / error callbacks with the parameters object as this would break the promise functionality
+
+The UI5 framework detects when the same OData call is done multiple times. It does only one request and returns the same result object for every call. If this result object is being changed, it changes for all calls. To prevent this from happening, the result is being copied with Object.assign()
+
+If you plan to use this with IE11, make sure to provide the [Object.assign Polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#polyfill).
+
+```js
+/** 
+ * Returns the result of the OData call
+ * @param {object} payload - The payload which equals to the entity
+ * @param {object} parameters - The parameters like Filters added to the OData call
+ * @returns {Promise} Promise object represents result of the OData call
+ */
+function readMyEntity(payload, parameters) {
+  var model = this.getView().getModel();
+  parameters = parameters || {};
+  return model.metadataLoaded()
+    .then(function () {
+      return new Promise(function (resolve, reject) {
+        var key = model.createKey("/MyEntitySet", payload)
+        // prevent success / error callbacks to be overwritten
+        var params = Object.assign({}, parameters,
+          {
+            success: function (data, response) {
+              // prevent accidently change of response data for subsequent calls
+              var dataCopy = Object.assign({}, data);
+              resolve({ data: dataCopy, response: response });
+            },
+            error: function (error) {
+              // additional error handling when needed
+              reject(error);
+            }
+          });
+        model.read(key, params);
+        // update could look like this:
+        // model.update(key, payload, params);
+        // create could look like this:
+        // model.create("/MyEntitySet", payload, params);
+        // delete could look like this:
+        // model.remove(key, params);
+      })
+    })
+}
+
+readMyEntity({ id: 12345 })
+  .then(function (result) {
+    console.log(result.data, result.response);
+  })
+  .catch(function (error) {
+    console.error(error);
+  });
+```
+
+### Chaining multiple OData calls together
+
+Breaks after the first Promise which does not resolve.  
+The finally method is not supported by the Promise Polyfill for IE11.
```

Comment:
sorry, again the code fencing thing: `finally`

Reply:
done in one of the latest commits